lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -965,14 +962,14 @@ std::pair<unsigned, bool> FVectorType::rootChildFieldID(unsigned fieldID,
 // CMemory Type
 //===----------------------------------------------------------------------===//
 
-void CMemoryType::print(mlir::DialectAsmPrinter &printer) const {
-  printer << "cmemory<";
+void CMemoryType::print(AsmPrinter &printer) const {
+  printer << "<";
```

Comment:
The "cmemory" string is now automatically printed, yes?

Reply:
Yes, mnemonic of type is now automatically printed.